src/Models/ArduinoDeviceBase.ts

```diff
@@ -273,16 +113,16 @@ export abstract class ArduinoDeviceBase implements Device, LibraryManageable {
   }
 
   async manageLibrary(): Promise<boolean> {
-    try {
-      // const libraryList;
-      await this.docker
-        .command(`run --name ${this.containerName} -v ${
-          this.arduinoPackagePath}:/root/ ${this.imageName} lib search`).then((data) => {
+    // try {
+    //   // const libraryList;
+    //   await this.docker
+    //     .command(`run --name ${this.containerName} -v ${
+    //       this.arduinoPackagePath}:/root/ ${this.imageName} lib search`).then((data) => {
 
-          });
-    } catch (error) {
-      throw error;
-    }
+    //       });
+    // } catch (error) {
+    //   throw error;
+    // }
     return true;
   }

```

Comment:
Always return true?
If so there is nothing to do with the LibraryManageable interface

Reply:
Will use TODO tag.